debs/SPECS/wazuh-manager/debian/postinst

```diff
@@ -15,6 +15,7 @@ case "$1" in
     OSMYSHELL="/sbin/nologin"
     SCRIPTS_DIR="${WAZUH_GLOBAL_TMP_DIR}/manager_installation_scripts"
     SCA_BASE_DIR="${SCRIPTS_DIR}/sca"
+    VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
```

Comment:
We should find the best way to manager the file versioning.

Reply:
For the moment it will be handled like this.

Reply:
Ok, thank you